guides/attachvolume.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Setup a Volume'
+icon: 'database'
+iconType: 'solid'
+---
+
+Volumes provide persistent block storage that can be attached to instances for storing data independently of the instance lifecycle. This makes them ideal for scenarios where you need to maintain critical data even after instances are deleted or replaced. Volumes can be reused and mounted across multiple instances, providing flexibility and durability for your storage needs.
+
+In this guide, we will cover how to create a volume, attach it to an instance, and verify that it is mounted via the Shadeform API. This process involves creating a volume first, launching an instance with the volume attached, and confirming the volume on the instance.
+
```

Comment:
Can we add a line here that says "Currently, volumes can only be attached to instances in the same cloud provider and region. Multi-cloud volumes are an upcoming feature."